README.md

```diff
@@ -1 +1,36 @@
-# Review-portfolio
\ No newline at end of file
+# Portfolio
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio
+
+> this project is the second project for the full stack web development class by microverse.
+
+## Built With
+
+- HTML
+- CSS
+
+## Live Demo (if available)
```

Comment:
Please remove the "(if available)" text from the readme file. Since there is already a live preview no need to show the "(is available)" content.